clouseau/__init__.py

```diff
@@ -0,0 +1,8 @@
+import os
+import libmozdata.config as config
+
+
+here = os.path.abspath(__file__)
+path = os.path.join(here, '../config.ini')
```

Comment:
We can use os.path.dirname instead of creating a path containing '..'.

We should also make libmozdata check for `config.ini` in several directories (home directory first, then current working directory, etc.).